docs/semgrep-ci/network-broker.md

```diff
@@ -80,15 +80,15 @@ Ensure that you are logged in to the server where you want to run Semgrep Networ
       ...
   ```
 
-1. Add the generated public key to the Semgrep Cloud Platform:
+1. Add your public key generated in step 3 to the Semgrep Cloud Platform:
```

Comment:
```suggestion
1. Add the public key that you generated in a previous step to Semgrep Cloud Platform:
```

Reply:
Updated by removing reference to previous step.